clippy.toml

```diff
@@ -0,0 +1,12 @@
+disallowed-methods = [
+    { path = "std::thread::Builder::spawn", reason = "Wrapper function should be used instead, refer to https://github.com/tikv/tikv/pull/12442 for more details." },
```

Comment:
Better explain it directly. And should mention what `wrapper` is needed exactly.